tests/unit/convertSelectorsToCSS.ts

```diff
@@ -0,0 +1,26 @@
+import convertSelectorsToCSS from '../../src/convertSelectorsToCSS';
+
+const { describe, it } = intern.getInterface('bdd');
+const { assert } = intern.getPlugin('chai');
+
+describe('Converting selectors to CSS', () => {
+	it('Converts a single selector to CSS', () => {
+		const css = convertSelectorsToCSS(['selector-1']);
+		assert.equal(css, '.selector-1 {}');
+	});
+
+	it('Converts many selectors to CSS', () => {
+		const css = convertSelectorsToCSS(['selector-1', '  selector-2  ', 'selector-3']);
+		assert.equal(css, '.selector-1 {}\n\n.selector-2 {}\n\n.selector-3 {}');
+	});
+
+	it('Removes the _key selector', () => {
+		const css = convertSelectorsToCSS(['key', '_key', ' key_', '_key_']);
```

Comment:
you adjusted the spacing on the wrong ` _key`

Reply:
Heh good catch, fixed!